app/views/email_authentications/passwords/expired_reset_token.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="error-body">
+  <div class="dialog">
+    <div class="text">
+      <img src="http://simple.org/images/simple-wordmark-standard.svg">
+      <h1>Reset link expired</h1>
+      <p>
+        The password link expired. <%= link_to "Click here", new_email_authentication_password_path %> to request a new link.
+      </p>
+    </div>
+    <%= link_to "RETURN HOME", "/" %>
+  </div>
+</div>
```

Comment:
I took a stab at re-using our existing components and styles for this page.

```erb
<div class="col-md-6 mx-auto">
    <div class="card">
        <div class="row">
            <div class="col">
                <h1>Reset link expired</h1>
            </div>
            <div class="col text-right login-logo">
                <%= logo_for_environment %>
            </div>
        </div>
        <div class="row">
          <div class="col">
            <p>
              The password link expired. <%= link_to "Click here", new_email_authentication_password_path %> to request a new link.
            </p>
          </div>
        </div>
    </div>
    <div class="actions mt-4" style="line-height: 2.5em;">
      <%= link_to "Return to home", "/" %>
    </div>
</div>
```

Fancy taking this for a spin? If you like it, we can remove the custom styles we've added. There may be more available components (either from Bootstrap or from our existing styles) that we can use. I'd run this by Claudio to get his 👍 before we ship a final version.

Reply:
I met with Claudio about this and he wanted to keep the 404 styling in the app, so we've worked on it and tried to match the 404 page as closely as could with alongside some changes he requested.